UI/Selenium/Steps/HearingScheduleSteps.cs

```diff
@@ -33,7 +36,16 @@ public void GivenTheHearingHasTheFollowingScheduleDetails(Table table)
 
         private void EnterHearingSchedule(HearingSchedule hearingSchedule)
         {
-            Driver.FindElement(HearingSchedulePage.HearingDate).SendKeys(hearingSchedule.HearingDate.FirstOrDefault().ToString("dd/MM/yyyy"));
+            try
+            {
+                IJavaScriptExecutor js = (IJavaScriptExecutor)Driver;
+                js.ExecuteScript("document.getElementById('hearingDate').setAttribute('type', '')");
```

Comment:
What is this for?  We have been fine without it so far.

Reply:
this fix need to be there for firefox and edge as the code is not typing in date field and later same day date is throwing validation error.

Reply:
All right.  So the best way is to make this happen **only** for Firefox and if not possible then checks with every other browser to ensure that this does not cause issues there.